cycode/cli/code_scanner.py

```diff
@@ -336,7 +337,19 @@ def scan_disk_files(context: click.Context, path: str, files_to_scan: List[str])
         if not content:
             continue
 
-        documents.append(Document(file, content, is_git_diff))
+        file_name = file
+
+        if _is_iac(scan_type) and _is_tfplan_json_file(file, content):
+            try:
+                content = tf_content_generator.generate_tf_content_from_tfplan(content)
```

Comment:
mb raise TfplanKeyError inside generate_tf_content_from_tfplan instead of catching KeyError in the root?